components/Common/Dropdown/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import type { DropdownItem } from '../../../types';
+import styles from './index.module.scss';
+
+export interface DropdownProps {
+  items: Array<DropdownItem>;
+  shouldShow: boolean;
+  styles: Object;
+}
+
+const Dropdown = ({ items, shouldShow, styles: css }: DropdownProps) => {
+  const mappedElements =
+    items &&
+    items.map(item => {
+      const extraStyles = {
+        fontWeight: item.active ? 'bold' : 'normal',
+      };
+      const handleKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
```suggestion

      const handleKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
```

Reply:
@ovflowd did not understand the suggested change here. can you elaborate?

Reply:
Just a new empty line to separate the extra styles and the function 🙇